ts/store/reducers/index.ts

```diff
@@ -194,6 +195,16 @@ export function createRootReducer(
                 // eslint-disable-next-line no-underscore-dangle
                 _persist: state.features.whatsNew._persist
               },
+              loginFeatures: {
```

Comment:
What's the difference between login and authentication?
Why we need a "loginFeatures" wrapper?
Why we need a fastLogin and a optIn wrapper just to store a boolean?